.github/workflows/deploy.yml

```diff
@@ -0,0 +1,35 @@
+name: Interactive Book Deployment
+
+on:
+  push:
+    branches:
+      - GSoD2020
+
+jobs:
+  github-pages:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      
+      # Use GitHub Actions' cache to shorten build times and decrease load on servers
+      - uses: actions/cache@v2
+        with:
+          path: vendor/bundle
+          key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile') }}
+          restore-keys: |
+            ${{ runner.os }}-gems-
+
+      - name: Build
+        uses: helaili/jekyll-action@v2

```

Comment:
We can't use this plugin to deploy as in future we need to add a step for Python script to generate APIs.

Reply:
jekyl serve is anyway a separate action and has nothing to do with jekyl build right? So we can use this step independently.

Reply:
True. Let's deploy using only this action. We will keep the python script stuff in a different workflow. Sounds Good ?